### DISCUSSION TITLE
Hai March Festival expiry dates

### DISCUSSION THREAD

**Discussion by Ricky-Blackman** [endless-sky/endless-sky#10141] [Balance, Mechanics, and Interactions]
The Hai have a festival in March, which generates a bunch of lucrative missions for the month of March in Hai space.
However, they don't have an expiry, so the player can simply travel from planet to planet, loading up on festival missions. Then, once March ends, he can carry out those missions. Of course, he has to maintain sufficient bunk space in order to follow this strategy.

This seems a bit OP to me. I suggest giving all of these missions an expiry date of 31 March. It also makes the narrative more realistic - delivering passengers hoping to celebrate a March festival in May seems a little odd.

It's a very small issue, but as a proud pedant (it is Pride month, I can come out as pedantic) I can't help but notice it. I have done a couple of brief searches for this; I apologize if it has already been raised.

**Comment by Tangle10** [comment]
Isn't this intended? There's a little extra line if you deliver late.

**Reply by ziproot** [reply]
There is?

**Comment by ziproot** [comment]
@qangin https://github.com/endless-sky/endless-sky/commit/c88386d6d49244c4290d9348dc04f6c023def472

**Reply by Ricky-Blackman** [reply]
Legend! I don't get this sort of service when I complain about significant issues with software and systems that I pay for!
